### ISSUE TITLE
GUI for displaying discards

### ISSUE THREAD

**Issue by dperelman** [alpacafondue/tts_crew#5]
The BoardGameArena version has a nice compact display of which cards have been discarded/still in play. It's basically just a list of numbers 1-9 in each color and 1-4 in black that change saturation to indicate the corresponding card has been played. In this mod, using search on the discard pile gives that information but not as nicely.

While I haven't done Lua or TTS mod development before, this seems like a good small project that I could probably figure out how to do myself, so I'll try to make a PR for this soon, but it might take me a bit to get up to speed.

**Comment by tableaukun** [comment]
Technically, if I'm remembering the rules correctly, you're only allowed to look at the most recently discarded trick. All previous tricks shouldn't be searchable, but I can see where the option could add some flexibility.

**Comment by alpacafondue** [comment]
Technically, if I'm remembering the rules correctly, you're only allowed to look at the most recently discarded trick. All previous tricks shouldn't be searchable, but I can see where the option could add some flexibility.

**Comment by dperelman** [comment]
You are right, the rules do say that (near the top of page 4), so I guess allowing viewing the discard is a house rule we like to play with, since we don't want to make it a memory game. (When we play in person, we sometimes organize the cards face-up to the side.)

**Comment by jChapman** [comment]
I think it would be nice to (optionally?) have a display of the last completed trick

**Comment by dperelman** [comment]
Should be straightforward to highlight the most recent trick in a different color than the other discarded cards.

**Comment by jChapman** [comment]
I was thinking only showing the last trick (as per the rules). Might grab some of your code to make it happen :)

**Comment by dperelman** [comment]
I just pushed 82901e353dc62cb8d9acb98ebd406579d385effb which makes my discard display highlight the most recent trick. If you want the display as a grid like that, you could probably modify the code to not show information about old discards (easiest would probably be to just make all of the numbers gray (`#444444`) at the start of the round.

**Comment by jChapman** [comment]
I added a UI element above Sort Hand 
![image](https://user-images.githubusercontent.com/1152464/132414521-e1fad845-ae8e-43dd-a0e2-601976a4f5c3.png)

See my changes on https://github.com/jChapman/tts_crew/commit/a5f09e1eda7d80afa8ad3432c4abfe0bc37e4c50
